python/lsst/cp/pipe/ptc.py

```diff
@@ -171,32 +160,27 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         doc="Use bootstrap for the PTC fit parameters and errors?.",
         default=False,
     )
-    maxAduForLookupTableLinearizer = pexConfig.Field(
-        dtype=int,
-        doc="Maximum DN value for the LookupTable linearizer.",
-        default=2**18,
-    )
     instrumentName = pexConfig.Field(
         dtype=str,
         doc="Instrument name.",
         default='',
     )
 
 
-@dataclass
-class LinearityResidualsAndLinearizersDataset:
-    """A simple class to hold the output from the
-       `calculateLinearityResidualAndLinearizers` function.
-    """
-    # Normalized coefficients for polynomial NL correction
-    polynomialLinearizerCoefficients: list
-    # Normalized coefficient for quadratic polynomial NL correction (c0)
-    quadraticPolynomialLinearizerCoefficient: float
-    # LUT array row for the amplifier at hand
-    linearizerTableRow: list
-    meanSignalVsTimePolyFitPars: list
-    meanSignalVsTimePolyFitParsErr: list
-    meanSignalVsTimePolyFitReducedChiSq: float
+# @dataclass
+# class LinearityResidualsAndLinearizersDataset:
+#     """A simple class to hold the output from the
+#        `calculateLinearityResidualAndLinearizers` function.
+#     """
+#     # Normalized coefficients for polynomial NL correction
+#     polynomialLinearizerCoefficients: list
+#     # Normalized coefficient for quadratic polynomial NL correction (c0)
+#     quadraticPolynomialLinearizerCoefficient: float
+#     # LUT array row for the amplifier at hand
+#     linearizerTableRow: list
+#     meanSignalVsTimePolyFitPars: list
+#     meanSignalVsTimePolyFitParsErr: list
+#     meanSignalVsTimePolyFitReducedChiSq: float
```

Comment:
I see that this dataclass is fully commented. If it is not being used anymore, then it should be completely removed.